requirements.txt

```diff
@@ -20,4 +20,4 @@ ninja
 # setuptools was removed from default python install
 setuptools ; python_version >= "3.12"
 packaging
-optree>=0.12.0
+optree>=0.12.0 ; python_version <= "3.12"
```

Comment:
FYI @XuehaiPan reverting this as well then as we cannot install it for 3.13 binaries testing as it is only available as an sdist